Makefile

```diff
@@ -9,23 +9,23 @@ SIMULATOR_ARCHIVE_PATH = ./build/ios_simulator.xcarchive/Products/Library/Framew
 IOS_ARCHIVE_PATH = ./build/ios.xcarchive/Products/Library/Frameworks/
 
 setup:
-	(pod install)
-	(cd SampleApps/$(APP_NAME) && pod install)
+	(bundle exec pod install)
```

Comment:
Yeah, we'll probably need to separate the commands out to ci vs local commands. We can do that in a separate PR. I'm just doing the minimal possible to get CI to pass given the changes to Assurance.

Reply:
Also, the bundle exec commands still work locally (for me at least). It could be a good switch to bundle exec to ensure we're all running the sam eversion of cocoapods.

Reply:
It requires you to handle the bundler installed locally, which shouldn't be required in my opinion. Can we add bundle exec in the circleci config or update the makefile for ci? 
@kevinlind what do you think?

Reply:
Updated the CI script with CI only commands.

Reply:
I see Nick already updated the Makefile, so that's fine. Don't need to change it now. Though, I  would be okay with investigating using bundle exec in the Makefile. The setup target could verify the bundler is installed and install it if needed. I do find it annoying when we get out of sync with our CocoaPods versions and `make setup` causes a diff.